test/unit/stream_alert_rule_processor/test_helpers.py

```diff
@@ -0,0 +1,65 @@
+'''
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+from mock import Mock
+
+from unit.stream_alert_rule_processor import (
+    REGION,
+    FUNCTION_NAME
+)
+
+
+def _get_mock_context():
+    """Create a fake context object using Mock"""
+    arn = 'arn:aws:lambda:{}:123456789012:function:{}:development'
+    context = Mock(invoked_function_arn=(arn.format(REGION, FUNCTION_NAME)),
+                   function_name=FUNCTION_NAME)
+
+    return context
+
+
+def _get_valid_config():
+    """Helper function to return a config that is valid
```

Comment:
specify what type of config it's returning - the dict below is actually non representative of a full config